tools/test-proxy/documentation/asset-sync/README.md

```diff
@@ -27,7 +25,7 @@ An `assets.json` takes the form:
   "AssetsRepo": "Azure/azure-sdk-assets-integration",
   "AssetsRepoPrefixPath": "python",
   "TagPrefix": "python/core",
-  "Tag": "python/core<Guid>"
+  "Tag": "python/core<10-character-commit-SHA>"
```

Comment:
Nit: The Tag is <TagPrefix>_<First 10 characters of the commit sha>. There's an underscore in between them.